content/docs/command-reference/add.md

```diff
@@ -148,6 +165,17 @@ not.
   > Note that external outputs typically require an external cache setup. See
   > link above for more details.
 
+- `--to-remote` - Track an external target, but don't move it into the
+  workspace, nor cache it. [Transfer](#transferring-data-directly-to-the-remote)
+  it directly to remote storage instead. Use `dvc pull` to get the data locally.
```

Comment:
> Track an external target

So does this option have any effect or throw an error if `--external` isn't used @isidentical ? Maybe we need an example after all.

Reply:
Yeah I just tried this and `--external` seems to be needed. Should be mentioned here?

Reply:
What do you mean by `--external` is needed?

Reply:
I tried `dvc add --to-remote /external/path` (a default remote exists) and it failed. You have to combine `--external` too right? Unless I did something wrong. If I'm right, let's mention that requirement here ^ and maybe we should consider automatically applying --external in --to-remote so there's no need to type both?